pkg/lib/operatorstatus/csv_reporter.go

```diff
@@ -80,7 +84,7 @@ func (r *csvStatusReporter) GetNewStatus(existing *configv1.ClusterOperatorStatu
 
 		if context.WorkingToward == nil {
 			builder.WithProgressing(configv1.ConditionFalse, fmt.Sprintf("Uninstalled version %s", csv.Spec.Version)).
-				WithAvailable(configv1.ConditionFalse, "")
+				WithAvailable(configv1.ConditionFalse, "", "")
```

Comment:
These other two callsites should set an informative message, too.

Reply:
@benluddy updated this calling site, but for the other calling site, i.e when the co is being initialized and is waiting for events in the CSV, looks like the Progressing condition has the skeleton message "waiting for events - source=csv.Name". That should be enough right? Coz the Available condition will probably have to spew out the same message otherwise. Also, since this in only during init, I'm guessing no one's getting paged when the cluster is being created/upgraded.